exercises/strings/strings00.cairo

```diff
@@ -0,0 +1,25 @@
+%lang starknet
+
+# Cairo supports short strings which are encoded as ASCII under the hood
```

Comment:
I think adding a link to a corresponding doc could be useful, if it exists :)

Reply:
Like for some other sections you can add a `README.md` file in the `strings` folder to point out to some doc or give more in-depth explanations about strings.